translations/monero-core_lt.ts

```diff
@@ -0,0 +1,2781 @@
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="290"/>
+        <source>Address</source>
+        <translation>Adresas</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="319"/>
+        <source>Please choose a file to sign</source>
+        <translation>Pasirinkite bylą, kurią norite pasirašyti</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="211"/>
+        <location filename="../pages/Sign.qml" line="234"/>
+        <location filename="../pages/Sign.qml" line="274"/>
+        <source>Verify</source>
+        <translation>Patikrinti</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="330"/>
+        <source>Please choose a file to verify</source>
+        <translation>Pasirinkite bylą, kurią norite patikrinti</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="198"/>
+        <location filename="../pages/Sign.qml" line="199"/>
+        <location filename="../pages/Sign.qml" line="304"/>
+        <location filename="../pages/Sign.qml" line="309"/>
+        <source>Signature</source>
+        <translation>Parašas</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="222"/>
+        <source>Message to verify</source>
+        <translation>Žinutė patikrinimui</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="249"/>
+        <source>Filename with message to verify</source>
+        <translation>Bylos pavadinimas su žinute, kuriuos reikia patikrinti</translation>
+    </message>
+</context>
+<context>
+    <name>StandardDialog</name>
+    <message>
+        <location filename="../components/StandardDialog.qml" line="133"/>
+        <source>Double tap to copy</source>
+        <translation>Spauskite du kartus, kad nukopijuoti</translation>
+    </message>
+    <message>
+        <location filename="../components/StandardDialog.qml" line="140"/>
+        <source>Content copied to clipboard</source>
+        <translation>Turinys nukopijuotas</translation>
+    </message>
+    <message>
+        <location filename="../components/StandardDialog.qml" line="154"/>
+        <source>Cancel</source>
+        <translation>Atšaukti</translation>
+    </message>
+    <message>
+        <location filename="../components/StandardDialog.qml" line="163"/>
+        <source>OK</source>
+        <translation>GERAI</translation>
+    </message>
+</context>
+<context>
+    <name>StandardDropdown</name>
+    <message>
+        <location filename="../components/StandardDropdown.qml" line="163"/>
+        <source>Low (x1 fee)</source>
+        <translation>Žemas (x1 rinkliavos)</translation>
+    </message>
+    <message>
+        <location filename="../components/StandardDropdown.qml" line="164"/>
+        <source>Med
```

Comment:
What happened here?

Reply:
Fixed everything.
Hopefully squashed it right.
I'm hardly understand  'blackball' section so any terminology would help.